build/azDevOps/azure/air-infrastructure-data-vars.yml

```diff
@@ -34,7 +34,7 @@ variables:
   - name: tf_state_container
     value: tfstate
   - name: tf_state_key_dev
-    value: $(domain)-dev
+    value: $(domain)_dev
```

Comment:
Any reason it's `_dev` and `-prod`

Reply:
updated now, was doing some local testing